tests/helpers/create-glimmer-component.js

```diff
@@ -0,0 +1,9 @@
+import { getContext } from '@ember/test-helpers';
+import GlimmerComponentManager from '@glimmer/component/-private/ember-component-manager';
+
+export default function createComponent(lookupPath, named = {}) {
+  const { owner } = getContext();
+  const { class: componentClass } = owner.factoryFor(lookupPath);
+  const componentManager = new GlimmerComponentManager(owner);
+  return componentManager.createComponent(componentClass, { named });
+}
```

Comment:
On ne peut pas tester unitairement un composant glimmer en appelant `this.owner.factoryFor('component:my-component').create();`.
Plus d'info sur ce helper : https://timgthomas.com/